src/main/resources/fess_config.properties

```diff
@@ -517,11 +510,9 @@ page.web.auth.max.fetch.size=100
 page.path.mapping.max.fetch.size=1000
 page.request.header.max.fetch.size=1000
 page.scheduled.job.max.fetch.size=100
-page.search.field.log.max.fetch.size=100
 page.elevate.word.max.fetch.size=1000
 page.bad.word.max.fetch.size=1000
 page.dictionary.max.fetch.size=1000
-page.relatedcontent.max.fetch.size=5000
```

Comment:
Do not remove it. See https://github.com/codelibs/fess/commit/4b337f602c14c351feb37817443a1c9c54f5e23f